app/services/platforms/youtube.ts

```diff
@@ -150,25 +157,28 @@ export class YoutubeService extends Service implements IPlatformService {
   }
 
   @requiresToken()
-  putChannelInfo(streamTitle: string, streamGame: string): Promise<boolean> {
-    const headers = new Headers();
-    headers.append('Content-Type', 'application/json');
-
-    const data = { snippet: { title: streamTitle }, id: this.liveStreamId };
-    const endpoint = 'liveBroadcasts?part=snippet';
-
-    const request = new Request(
-      `${this.apiBase}/${endpoint}&access_token=${this.oauthToken}`,
-      {
-        method: 'PUT',
-        headers,
-        body: JSON.stringify(data)
-      }
-    );
+  putChannelInfo(streamTitle: string, streamDescription: string): Promise<boolean> {
+    return this.fetchDescriptionString().then(autopublishString => {
+      const headers = new Headers();
+      headers.append('Content-Type', 'application/json');
+
+      const fullDescription = autopublishString.concat(streamDescription);
```

Comment:
Is a simple concatenation enough?  Should we be adding a newline or something?  Not really sure.

Reply:
The autopublish string has a newline in it if enabled.